examples/config/config

```diff
@@ -258,14 +260,20 @@ set history_disable_easter_egg 1
 @cbind  w            = event REQ_NEW_WINDOW
 
 # Page movement binds
-@cbind  j            = scroll vertical 20
-@cbind  k            = scroll vertical -20
-@cbind  h            = scroll horizontal -20
-@cbind  l            = scroll horizontal 20
+@cbind  j            = scroll vertical 100
+@cbind  J            = scroll vertical 50
```

Comment:
Logically it just depends how you look at it. Would read: `Shift` to *increase precision* (same in visual tools like **blender** for example)

Technically it's just a question of comfort: I found scrolling by just 20 too tedious (even though my key rate is set to high).

So the default for me should be a jump that allows for quick scrolling while still allowing for getting a more focussed view when needed.

Anyway, in order to estimate the statistical preferencea, we'd need some more people in the discussion. Until then we're "just two opinions", so we might just leave the setting as-is.